Description: Added by Kubuntu for update-notifier-kde to prompt for restricted codec install
Origin: vendor
Forwaded: not-needed
Author: Harald Sitter <apachelogger@ubuntu.com>
Index: kde-baseapps-4.9.1/konqueror/src/konqmainwindow.cpp
===================================================================
--- kde-baseapps-4.9.1.orig/konqueror/src/konqmainwindow.cpp	2012-09-04 20:47:44.694999847 +0530
+++ kde-baseapps-4.9.1/konqueror/src/konqmainwindow.cpp	2012-09-04 20:48:35.887999847 +0530
@@ -202,6 +202,17 @@
     , m_pURLCompletion(0)
     , m_isPopupWithProxyWindow(false)
 {
+  //Added by Kubuntu for update-notifier-kde to prompt for restricted codec install
+  QDBusMessage m = QDBusMessage::createMethodCall(QLatin1String("org.kubuntu.restrictedInstall"),
+                                                  QLatin1String("/org/kubuntu/restrictedInstall"),
+             QLatin1String("org.kubuntu.restrictedInstall"),
+             QLatin1String("installRestricted"));
+  QList<QVariant> dbusargs;
+  dbusargs.append(KonqFactory::aboutData()->programName());
+  dbusargs.append(QLatin1String("flashplugin-installer"));
+  m.setArguments(dbusargs);
+  bool queued = QDBusConnection::sessionBus().send(m);
+
   incInstancesCount();
   setPreloadedFlag( false );
 
